storage/remote/storage.go

```diff
@@ -120,6 +133,30 @@ func (s *Storage) ApplyConfig(conf *config.Config) error {
 	return nil
 }
 
+func (s *Storage) newQueue(i int, rwConf *config.RemoteWriteConfig, externalLabels model.LabelSet) (*QueueManager, error) {
+	c, err := NewClient(i, &ClientConfig{
```

Comment:
In keeping with the style of this file, can we use long-form variable names (like `client` in this case)?